README.md

```diff
@@ -84,6 +117,10 @@ Building (and running) is continuously tested under Linux, macOS, and Windows us
 
 If you use our tool for your research, we will be thankful if you refer to it by citing the appropriate publication:
 
+T. Grurl, C. Pichler, J. Fuss and R. Wille, "Automatic Implementation and Evaluation of Error-Correcting Codes for
+Quantum Computing: An Open-Source Framework for Quantum Error-Correction," in International Conference on VLSI
+Design and International Conference on Embedded Systems (VLSID), 2023
```

Comment:
Is this really the proper name for the conference? Shouldn't the "and ..." part be removed?

Reply:
According to IEEE it is.